### PR TITLE
details for Kyverno projects

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

### CODE REVIEW

File: summerofcode/2022.md [hunk 1]
```diff
@@ -387,32 +387,32 @@ If you are a project maintainer and consider mentoring during the GSoC 2022 cycl
...
 
-- Description: Kyverno policies can validate, mutate, and generate any Kubernetes resource. This feature enables time-based policies for Kyverno to allow select operations for authorized users based on time ranges and schedule constraints.
-- Expected outcome:
+- Description: Kyverno policies can validate, mutate, and generate any Kubernetes resource. A common use case is for organizations to allow access to Kubernetes cluster resources based on schedules, such as a pager rotation schedule for SREs. This feature enables time-based policies for Kyverno, to allow select operations for authorized users based on time ranges and schedule constraints. 
+- Expected outcome: The feature will extend Kyverno's policy rule definition to support time ranges and block requests which do not match the configured values.
 - Recommended Skills: Golang, Kubernetes
 - Mentor(s): Chip Zoller (@chipzoller), Jim Bugwadia (@JimBugwadia)
-- Expected project size: 175 or 350 Hours
-- Difficulty: Easy, Medium, or Hard
+- Expected project size: 200 hours
```
Comment (nate-double-u) [inline summerofcode/2022.md:404]: Please use 175 or 350 hours.

File: summerofcode/2022.md [hunk 2]
```diff
@@ -387,32 +387,32 @@ If you are a project maintainer and consider mentoring during the GSoC 2022 cycl
...
-- Expected outcome:
+- Description: Kyverno, the Kubernetes native policy engine, is designed to easily secure and automate Kubernetes configurations. Kyverno uses an extended [JMESPath query language](https://jmespath.org/specification.html) for complex JSON data processing. Kyverno extends JMESPath by allowing nested expressions and an extended set of custom functions. As Kyverno's declarative policy language has evolved its become important to provide proper tooling around embedding, syntax checking and validation of JMESPath expressions including the extensions.
+- Expected outcome:  This project will define a formal specification of Kyverno's JMESPath extensions using ABNF notation and create a parser-validator for this grammar for use in Kyverno. 
 - Recommended Skills: Golang, Kubernetes, Language Design, DSLs
 - Mentor(s): Jim Bugwadia (@JimBugwadia)
-- Expected project size: 175 or 350 Hours
-- Difficulty: Easy, Medium, or Hard
+- Expected project size: 250 hours
```
Comment (nate-double-u) [inline summerofcode/2022.md:394]: Please use 175 or 350 hours.

File: summerofcode/2022.md [hunk 3]
```diff
@@ -387,32 +387,32 @@ If you are a project maintainer and consider mentoring during the GSoC 2022 cycl
...
 #### Dynamic approvals for Kubernetes resources
 
-- Description: Kyverno is a Kubernetes native policy engine that makes it easy to validate, mutate, and generate resources. This feature extends Kyverno to allow administrative approvals for certain requests allowing easier collaboration across developers and operations teams.
+- Description: Kyverno is a Kubernetes native policy engine that makes it easy to validate, mutate, and generate resources. While Kyverno policies allow configuratrion of exceptions, excluding policy enforcement on select resources requires manual configuration by cluster administrators. This feature extends Kyverno to allow users to request policy exceptions, and for administrative approvals to be granted asynchronously. The goal of this feature is to allow easier collaboration across developers and operations teams, and to make it easier to manage policies at scale.
 - Expected outcome:
 - Recommended Skills:  Golang, Kubernetes
 - Mentor(s): Shuting Zhao (@realshuting), Jim Bugwadia (@JimBugwadia)
-- Expected project size: 175 or 350 Hours
-- Difficulty: Easy, Medium, or Hard
+- Expected project size: 240 hours
```
Comment (nate-double-u) [inline summerofcode/2022.md:414]: Please use 175 or 350 hours.

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Thanks for the updates @JimBugwadia, could you please revise and use 175 or 350 hours.

**Comment by JimBugwadia** [comment]
Hi @nate-double-u - done! Thanks!

**Review by nate-double-u — APPROVED** [review]
Thanks for the updates @JimBugwadia

**Pull request merged**